inst/scribble/scribble.css

```diff
@@ -0,0 +1,162 @@
+:root {
+  --xe-scribble--pen-color: #FF0000;
+  --xe-scribble--button-draw-active-bg: #FF0000;
+  --xe-scribble--button-draw-active-fg: #FFFFFF;
+  --xe-scribble--button-active-color: #000000;
+  --xe-scribble--button-inactive-color: #666666;
+}
+
+.xe-scribble__cursor__eraser {
+  position: absolute;
+  height: 10px;
+  width: 10px;
+  border-radius: 50%;
+  pointer-events: none;
+  z-index: 5;
+}
+
+.xe-scribble__cursor__eraser.hidden {
+  display: none;
+}
+
+.xe-scribble {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+}
+
+.xe-scribble__canvas__drawing,
+.xe-scribble__canvas,
+.xe-scribble {
+  pointer-events: none;
+  z-index: 1;
+}
+
+.xe-scribble__canvas__drawing.active {
+  pointer-events: all;
+}
+
+.xe-scribble__tools {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  pointer-events: all;
+  z-index: 101;
+  width: 32px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(4, 1fr);
+  align-content: center;
+  justify-items: center;
+  grid-gap: 5px;
+}
+
+.xe-scribble__button {
+  cursor: pointer;
+  padding: 3px;
+  background: unset;
+  border: unset;
+  border-radius: 100%;
+  width: 100%;
+  height: calc(100% - 3px);
+}
+
+.xe-scribble__button {
+  opacity: 1;
+  transition: transform ease-out 100ms, opacity ease-out 80ms;
+}
+
+.xe-scribble__button:hover {
+  background-color: #eee;
+}
+.xe-scribble__button.active {
+  background-color: #4443;
+}
+
+.xe-scribble__tools.minimized > :nth-child(2) {
+  opacity: 0;
+  transform: translateY(-100%);
+  transition: ease-in 100ms, opacity ease-in 80ms;
+  z-index: 0;
+  pointer-events: none;
+}
+
+.xe-scribble__tools.minimized > :nth-child(3) {
+  opacity: 0;
+  transform: translateY(-200%);
+  transition: ease-in 100ms, opacity ease-in 80ms;
+  z-index: 0;
+  pointer-events: none;
+}
+
+.xe-scribble__tools.minimized > :nth-child(4) {
+  opacity: 0;
+  transform: translateY(-300%);
+  transition: ease-in 100ms, opacity ease-in 80ms;
+  z-index: 0;
+  pointer-events: none;
+}
+
+.xe-scribble__button > svg {
+  width: 100%;
+  height: 100%;
+}
+
+.xe-scribble__button__draw,
+.xe-scribble__button__erase,
+.xe-scribble__button__clear {
+  color: var(--xe-scribble--button-inactive-color);
+}
+
+.xe-scribble__button__draw:hover,
+.xe-scribble__button__erase:hover,
+.xe-scribble__button__clear:hover {
+  color: var(--xe-scribble--button-active-color);
+}
+
+.xe-scribble__button__draw.active {
+  color: var(--xe-scribble--button-draw-active-fg);
+  background-color: var(--xe-scribble--button-draw-active-bg);
+}
+
+.xe-scribble__button__erase.active > svg,
+.xe-scribble__button__clear:hover > svg {
+  color: var(--xe-scribble--button-active-color);
+}
+
+#colorPicker {
+  border-radius: 100%;
+  height: 22px;
+  width: 22px;
+  margin: 0;
+}
+
+#colorPicker {
+  border-radius: 100%;
+  height: 30px;
+  width: 30px;
+  border: none;
+  outline: none;
+  -webkit-appearance: none;
+}
```

Comment:
I think there is some redundancy here, these two ID definitions could probably be collapsed and consolidated. Also, should the color picker have a `#xe-scribble__colorPicker__` type ID or something? Just to avoid potential conflict.

Reply:
In the video you posted, the color picker was a circle, in my testing it was a square. Not sure if something changed in the last several commits that changed/broke this aesthetic.

Reply:
Yeah that would be a better id, I changed it to `#xe-scribble__colorPicker`. I poked the CSS a bit and now it should be a circle on Chrome too. I'm honestly not sure why it wasn't working before or why it's working now. CSS is weird.